gengo/gengo.py

```diff
@@ -448,7 +449,7 @@ def compatibletext(text):
     @staticmethod
     def unicode2utf8(text):
         try:
-            if isinstance(text, unicode):
+            if isinstance(text, str):
```

Comment:
I don't think we should change this. Because we need to call `encode()` when `unicode` object is passed with Python 2 to this method.
How about this?
```
PY3 = sys.version_info > (3, 0, 0)

@staticmethod
def unicode2utf8(text):
    if PY3:
        return text

    if isinstance(text, unicode):
        return text.encode('utf-8')
```